chef-utils/lib/chef-utils.rb

```diff
@@ -29,6 +29,7 @@
 require_relative "chef-utils/dsl/which"
 require_relative "chef-utils/dsl/windows"
 require_relative "chef-utils/mash"
+require_relative "chef-utils/dist"
```

Comment:
We can skip this part and continue to require dist when we need it within the specific ruby files